### ISSUE TITLE
SysVnVPlan FT1-TC3 issue #8

### ISSUE THREAD

**Issue by vajihehm** [PeaWagon/Kaplan#22]
@PeaWagon 
In “Test Input Geometry” It should be considered geometry values as you think purposefully bad or invalid. The example is not sufficient and you should specify a set of inputs as well as the output which is expected regarding to the given input.

![2](https://user-images.githubusercontent.com/42664092/47752928-4e1e7e00-dc6c-11e8-926d-6fd89a1b0237.png)

**Comment by smiths** [comment]
@PeaWagon, I agree with the feedback from @vajihehm - your test case should be more specific.  If the Basic Set Exchange is a good resource, you should say specifically what you are going to use from it.

**Comment by vajihehm** [comment]
@smiths Sure. Edited.

**Comment by smiths** [comment]
Thank you @vajihehm.
